app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::GreetingsController < ApplicationController
+  def index
+    render json: Greeting.order('RANDOM()').first
+  end
+end
```

Comment:
- [ ] Kindly fix the display of greetings, the react app only shows a "Hello world" text. See the images below for your reference.

![react](https://user-images.githubusercontent.com/53124548/177772371-8367848a-6904-41c9-947a-937d519d19fc.png)

### Messages from the API
![ruby](https://user-images.githubusercontent.com/53124548/177772719-9f4ebb51-3719-452a-a1c1-be76bff54a93.png)